contributors/devel/conformance-guidelines.md

```diff
@@ -0,0 +1,103 @@
+# Conformance Test Guidelines
+
+The Kubernetes conformance test suite is a set of testcases, currently a
+subset of the integration/e2e tests, that the Architecture SIG has approved
+to define the core set of interoperable features that all Kubernetes
+deployments must support.
+
+Contributors must write and submit e2e tests first (approved by owning Sigs). 
+Once the new tests prove to be stable in CI runs, later create a follow up PR 
+to add the test to conformance. This approach also decouples the production 
+of useful tests from their promotion to conformance.
+
+A conformance test verifies the expected functionality works as a user might encounter it in the wild, 
+and tests should begin by covering the most important and visible aspects of the function.
+
+Conformance tests in the default profile should cover features that are:
+
+- GA (not alpha or beta APIs, nor deprecated features)
+- portable (not dependent on provider-specific capabilities or on the public internet)
+- not an optional feature (e.g. not policy enforcement)
+- non-privileged (neither root on nodes, network, nor cluster)
+- cannot be flaky
```

Comment:
what does flaky mean in this context? pass N runs in a row? N% passes over M runs?  N% passes over M runs against a single commit?

Reply:
rewording to not mention as flaky, please suggest?
I have moved the Conformance sections from e2e testing document into this document and referred this doc in there.